apps/src/p5lab/utils.js

```diff
@@ -2,3 +2,18 @@
 export function randomInt(min, max) {
   return Math.floor(Math.random() * (max - min + 1)) + min;
 }
+
+// Requires two arrays
+export function arrayEquals(a, b) {
```

Comment:
Is it possible to instead use Underscore, maybe using [this](https://www.geeksforgeeks.org/underscore-js-_-isequal-function/)?

Reply:
It looks like this would work well. I'm not sure how we would want to go about adding the Underscore library to this file, however.

Reply:
I think you could just import Underscore into the caller and use its function instead of relying on this file?